docs/index.md

```diff
@@ -0,0 +1,62 @@
+
+# Getting started (napari)
+
+MIDI App Controller is a package designed to integrate MIDI controllers with Python Qt apps using app-model. As of now, it is used most commonly with [napari](napari.org), a viewer for multi-dimensional images. We will show how to use MIDI App Controller with napari but getting started with other applications should look very similar.
```

Comment:
The link to napari is broken. It directs to `https://midi-app-controller.readthedocs.io/en/add-docs/napari.org`.